gobblin-runtime/src/main/java/gobblin/runtime/util/JobStateToJsonConverter.java

```diff
@@ -87,6 +98,64 @@ public void convert(String jobName, String jobId, Writer writer)
       jsonWriter.close();
     }
   }
+  
+  /**
+   * Modify the configuration value of a given job instance.
+   */
+  public void modify(String jobName, String jobId, String key, String newValue) throws IOException {
+    List<? extends JobState> jobStates = this.jobStateStore.getAll(jobName, jobId + JOB_STATE_STORE_TABLE_SUFFIX);
+    if (jobStates.isEmpty()) {
+      LOGGER.warn(String.format("No job state found for job with name %s and id %s", jobName, jobId));
+      return;
+    }
+    JobState jobState = jobStates.get(0);
```

Comment:
Added `modifyAll` to convert all past 'JobState's`that have the same `jobName`. Isn't it the case that a pair of `jobName` and `jobId` maps to only one `JobState`?